src/App.js

```diff
@@ -14,27 +14,28 @@ function Home(){
       <ImageContainer />
       <Container />
       <ContainerDeal />
-      <ProductsContainer />
+      <OneCategoryProducts />
       <Footer/>
   </>
 }
 
+
+
 function GiftboxSection(){
    return <div className="gifts-section">
      <h1>Oops!! It seems like you have not logged in.</h1>
      <LogIn/>
      </div>
-   
 }
 
 
 function App() {
   return (
-  
     <div className="App">
      <Navbar /> 
+     <SearchContainer/>
       <Routes>
-        <Route path="/home" element={<Home />}/>
+        <Route path="/" element={<Home />}/>
```

Comment:
Here you can also create a route component and render it on app component, it will help app component to look more cleaner.